apps/spruce/src/constants/routes.ts

```diff
@@ -43,6 +43,8 @@ export enum ProjectSettingsTabRoutes {
   EventLog = "event-log",
   Containers = "containers",
   ViewsAndFilters = "views-and-filters",
+  AppSettings = "app-settings",
+  PermissionGroups = "permission-groups",
```

Comment:
Should we name these so that they indicate they are related to Github and so that they have the same names as `ProjectSettingsSection` values? It also makes the relation to github more apparent once we nest the tabs down the line.
```suggestion
  GithubAppSettings = "github-app-settings",
  GithubPermissionGroups = "github-permission-groups",
```